tests/spectron/alwaysOnTop.spectron.js

```diff
@@ -12,7 +12,7 @@ describe('Tests for Always on top', () => {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
 
     beforeAll((done) => {
-        return app.startApplication().then((startedApp) => {
+        return app.startApplication({alwaysOnTop: false}).then((startedApp) => {
```

Comment:
isn't this `false` by default?

Reply:
I've set it `true` by default since there are 2 command line windows overlapping Electron screen, so it is always necessary to set `true` for all tests except a test `alwaysOnTop`